### YamlMime:UniversalReference
api_name: []
items:
- children: []
  class: azure.iot.device.common.sastoken.SasTokenError
  fullName: azure.iot.device.common.sastoken.SasTokenError
  inheritance:
  - inheritance:
    - inheritance:
      - type: builtins.object
      type: builtins.BaseException
    type: builtins.Exception
  langs:
  - python
  module: azure.iot.device.common.sastoken
  name: SasTokenError
  source:
    id: SasTokenError
    path: azure-iot-device\azure\iot\device\common\sastoken.py
    remote:
      branch: master
      path: azure-iot-device\azure\iot\device\common\sastoken.py
      repo: https://github.com/Azure/azure-iot-sdk-python
    startLine: 15
  summary: Error in SasToken
  syntax:
    content: SasTokenError(message, cause=None)
  type: class
  uid: azure.iot.device.common.sastoken.SasTokenError
references: []
